readme.txt

```diff
@@ -36,6 +36,9 @@ Source: https://stocksnap.io/photo/4B83RD7BV9
 
 == Changelog ==
 
+= 3.4.1 - 2025-05-08 =
+* Fix: Critical Error after theme update [https://wordpress.org/support/topic/critical-error-on-website-54/#post-18454740]
```

Comment:
@nuritsha The markdown here seems to be missing the anchor text - is this on purpose?